pkg/volume/flexvolume/flexvolume.go

```diff
@@ -98,6 +98,10 @@ func (plugin *flexVolumePlugin) RequiresRemount() bool {
 	return false
 }
 
+func (plugin *flexVolumePlugin) SupportsMountOption() bool {
+	return false
```

Comment:
Agree, would be a nice to have mount options piped to Flex. But like @childsb mentioned, it'll require updating the interface. Would be nice to get it into 1.6. Otherwise in 1.7 we could extend the Flex mount command parameters to accept mount options. +@chakri-nelluri